omod/src/main/webapp/pages/conditionlist/addCondition.gsp

```diff
@@ -57,7 +57,7 @@
                         formFieldName: "conditionStartDate",
                         label        : "",
                         useTime      : false,
-                        endDate    : new Date(),
```

Comment:
Didn't we modify this in another pull request?

Reply:
Yes we did, but the changes were not merged to master when i started to work on this, so I made the change in this copy as well.

Reply:
It is not a good practice to mix issues from various tickets into one pull request. https://opensource.com/article/18/6/anatomy-perfect-pull-request